src/uu/ptx/src/ptx.rs

```diff
@@ -214,8 +213,22 @@ fn get_config(matches: &clap::ArgMatches) -> UResult<Config> {
         config.format = OutFormat::Roff;
         "[^ \t\n]+".clone_into(&mut config.context_regex);
     }
-    if matches.contains_id(options::SENTENCE_REGEXP) {
-        return Err(PtxError::NotImplemented("-S").into());
+    if let Some(regex) = matches.get_one::<String>(options::SENTENCE_REGEXP) {
+        config.sentence_regex = Some(regex.clone());
```

Comment:
can we avoid the clone() ?

Reply:
Thanks, avoided with `matches.remove_one`